packages/babel-highlight/src/index.ts

```diff
@@ -92,21 +93,21 @@ function highlightTokens(defs: Object, text: string) {
 }
 
 type Options = {
-  forceColor?: boolean,
+  forceColor?: boolean;
 };
 
 /**
  * Whether the code should be highlighted given the passed options.
  */
 export function shouldHighlight(options: Options): boolean {
-  return Chalk.supportsColor || options.forceColor;
+  return !!Chalk.supportsColor || options.forceColor;
 }
 
 /**
  * The Chalk instance that should be used given the passed options.
  */
 export function getChalk(options: Options) {
-  let chalk = Chalk;
+  let chalk: ChalkType = Chalk;
```

Comment:
TS should be able to infer this one correctly

Reply:
yes, but `Chalk`(default export of the modules) is more broad then `ChalkType`… and so later instance of `Chalk.constructor` would not be possible to assign to it

probably this can be refactored to something like 

`const chalk = options.forceColor ? … : …` from which correct type would be inferred, but I wanted to minimize changes here and postpone not strictly required refactoring to time when all codebase is in typescript

Reply:
Yeah this is a separate change and tracked by https://github.com/babel/babel/issues/12442